.travis.yml

```diff
@@ -37,54 +37,60 @@ submit_to_coverity_scan: &submit_to_coverity_scan
                "https://scan.coverity.com/builds";
   fi
 
-linux_gcc8: &linux_gcc8
+ubuntu1804_gcc10: &ubuntu1804_gcc10
   os: linux
-  dist: xenial
+  dist: bionic
   compiler: gcc
   addons:
     apt:
       update: true
-      sources: [ ubuntu-toolchain-r-test ]
-      packages: [ gcc-8, g++-8 ]
+      sources:
+        - sourceline: 'ppa:ubuntu-toolchain-r/test'
+      packages: [ gcc-10 g++-10 ]
   before_install:
-    - eval "export CC=gcc-8"
-    - eval "export CXX=g++-8"
+    - eval "export CC=gcc-10 CXX=g++-10"
     - eval "export COV_COMPTYPE=gcc COV_PLATFORM=linux64"
+    - eval "export BUILD_TOOL_OPTIONS='-j 4'"
+    - eval "export GENERATOR='Unix Makefiles'"
   install:
-    - *install_coverity
     - pip install conan --upgrade --user
 
-linux_clang: &linux_clang
+ubuntu1804_clang10: &ubuntu1804_clang10
   os: linux
-  dist: xenial
+  dist: bionic
   compiler: clang
   addons:
     apt:
       update: true
-      packages: [ ]
+      sources:
+        - sourceline: 'deb https://apt.llvm.org/bionic llvm-toolchain-bionic-10 main'
+          key_url: 'https://apt.llvm.org/llvm-snapshot.gpg.key'
+      packages: [ clang-10 clang++-10 ]
   before_install:
-    - eval "export CC=clang"
-    - eval "export CXX=clang++"
+    - eval "export CC=clang-10 CXX=clang++-10"
     - eval "export COV_COMPTYPE=clang COV_PLATFORM=linux64"
+    - eval "export BUILD_TOOL_OPTIONS='-j 4'"
+    - eval "export GENERATOR='Unix Makefiles'"
   install:
     - pip install conan --upgrade --user
 
-osx_xcode: &osx_xcode
+macos1015_xcode11_6: &macos1015_xcode11_6
   os: osx
-  osx_image: xcode11.1
+  osx_image: xcode11.6
   compiler: clang
   addons:
     homebrew:
-      packages: [ python3 ]
+      packages: [ pyenv-virtualenv ]
```

Comment:
This may also not exactly what I intended. `pyenv-virtualenv` probably has `python3` as a dependency... will fix this one too.